src/app/core/services/label.service.ts

```diff
@@ -55,22 +55,32 @@ export class LabelService {
   private static responseLabels: Label[] = Object.values(REQUIRED_LABELS.response);
   private static statusLabels: Label[] = Object.values(REQUIRED_LABELS.status);
   private static otherLabels: Label[] = Object.values(REQUIRED_LABELS.others);
-  private static labelArrays = {
+  private static teamOrModerationLabelArrays = {
```

Comment:
```suggestion
  private static allLabelArrays = {
```

Maybe we can adopt this naming since this encompasses most if not all the labels we have 
in our application. What are your thoughts? @ducbinh2611

Reply:
I am perfectly fine with it! The reason why I wanted to initially name it `teamOrModerationLabelArrays` was that I want to make it easy to understand which phase it belongs to. I think naming it as `allLabelArrays` is good also given the fact that we already name the other label array as `testerLabelArrays`